debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -208,7 +189,10 @@ public Lsn getMinLsn(String changeTableName) throws SQLException {
      * @throws SQLException
      */
     public void getChangesForTable(TableId tableId, Lsn fromLsn, Lsn toLsn, ResultSetConsumer consumer) throws SQLException {
-        final String query = GET_ALL_CHANGES_FOR_TABLE.replace(STATEMENTS_PLACEHOLDER, cdcNameForTable(tableId));
+        final String query = GET_ALL_CHANGES_FOR_TABLE
+                .replaceFirst(STATEMENTS_PLACEHOLDER,
+                        Matcher.quoteReplacement(sourceTimestampMode.lsnTimestampSelectStatement(supportsAtTimeZone)))
```

Comment:
Could this step be done once at set-up time? I think only the part with the table id  on the next line actually needs replacement upon invocation time of this method? Same for the method below.

Reply:
Done.